src/EditorFeatures/CSharp/CSharpEditorResources.resx

```diff
@@ -126,4 +126,7 @@
   <data name="Properties" xml:space="preserve">
     <value>Properties</value>
   </data>
+  <data name="UseAutoProperty" xml:space="preserve">
+    <value>Use auto property</value>
```

Comment:
Presumably there will be a VB analyzer, too?  If so, this string should be in `WorkspacesResources` so it can be reused.